src/pages/cli/usage/export-to-cdk.mdx

```diff
@@ -1,3 +1,76 @@
-## Export an Amplify backend as a CDK project
+export const meta = {
+  title: `Export Amplify project to CDK`,
+  description: `Export Amplify CLI-generated backends as a Cloud Development Kit (CDK) stack and incorporate into existing CDK deployment pipelines. This capability allows frontend developers to build their app backend quickly and, each time it is ready to ship, hand it over to DevOps teams to deploy to production.`,
+};
 
-## How to integrate an Amplify backend as part of a CDK pipeline
+Export Amplify CLI-generated backends as a Cloud Development Kit (CDK) stack and incorporate into existing CDK deployment pipelines. This capability allows frontend developers to build their app backend quickly and, each time it is ready to ship, hand it over to DevOps teams to deploy to production. 
+
+```bash
+amplify export --out <your-cdk-project-location> 
+```
+
+The command above exports your Amplify projects with CDK-compatible CloudFormation files and assets. 
+
+> Note: `notifications` category is not supported for export.
+
+In the exported location you should see a file structure like this:
+
+```bash
+amplify-export-myAmplifyProject/
+├── <YOUR_AMPLIFY_CATEGORIES>/
+│   ├──...
+│   ├──...
+│   └──...
+├── amplify-export-manifest.json
+├── category-stack-mapping.json
+├── export-tags.json
+└── root-stack-template.json
+```
+
+## Use an exported Amplify backend in AWS Cloud Development Kit (CDK)
+
+The export uses the current state of the Amplify backend to build, package, and generate the CloudFormation files and assets to the provided path. Read more in detail about the purpose of the files and structure generated https://quip-amazon.com/DYpUAaa3odlO/Amplify-CLI-Export-Docs#temp:C:JTdd2d3d79fe2794b87ba8d58808 section.
```

Comment:
Check the link @renebrandel

nit: "in detail" seems superfluous to me

```suggestion
The export uses the current state of the Amplify backend to build, package, and generate the CloudFormation files and assets to the provided path. Read more about the purpose of the files and structure generated https://quip-amazon.com/DYpUAaa3odlO/Amplify-CLI-Export-Docs#temp:C:JTdd2d3d79fe2794b87ba8d58808 section.
```

Reply:
```suggestion
The export uses the current state of the Amplify backend to build, package, and generate the CloudFormation files and assets to the provided path.```